docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -0,0 +1,108 @@
+
+To load data from a SQL Server database into Module Builder:
+
+1. In the data step of the Model Builder tool, select **SQL Server** from the data source dropdown.
+1. Select the button next to the **Connect to SQL Server database** text box.
+    1. In the **Choose Data** dialog, select **Microsoft SQL Server Database File**. 
+    1. Uncheck the **Always use this selection** checkbox and select **Continue**
+    1. In the **Connection Properties** dialog, select **Browse** and select the downloaded .MDF file.
+    1. Select **OK**
+1. Choose the data set name from the **Table Name** dropdown.
+1. From the **Column to Predict (Label)** dropdown, choose the data category on which you want to make a prediction.
+1. From the **Input Columns (Features)** dropdown, confirm the columns you want to include are checked. 
+
+You're done setting up your data source file for Model Builder. Select the **Train** link to move to the next step in the Model Builder tool.
+
+## Set up image data files
+
+Model Builder expects image data to be JPG files organized in folders that correspond to the categories of the classification. 
+
+To load images into Model Builder, provide the path to a single top-level directory:
+
+- This top-level directory contains one subfolder for each of the categories to predict. 
+- Each subfolder contains the image files belonging to its category. 
+ 
+In the folder structure illustrated below, the top-level directory is *flower_photos*. There are five subdirectories corresponding to the categories you want to predict: daisy, dandelion, roses, sunflowers, and tulips. Each of these subdirectories contains images belonging to its respective category. 
+
+```text
+\---flower_photos
+    +---daisy
+    |       100080576_f52e8ee070_n.jpg
+    |       102841525_bd6628ae3c.jpg
+    |       105806915_a9c13e2106_n.jpg
+    |       
+    +---dandelion
+    |       10443973_aeb97513fc_m.jpg
+    |       10683189_bd6e371b97.jpg
+    |       10919961_0af657c4e8.jpg
+    |       
+    +---roses
+    |       102501987_3cdb8e5394_n.jpg
+    |       110472418_87b6a3aa98_m.jpg
+    |       118974357_0faa23cce9_n.jpg
+    |       
+    +---sunflowers
+    |       127192624_afa3d9cb84.jpg
+    |       145303599_2627e23815_n.jpg
+    |       147804446_ef9244c8ce_m.jpg
+    |       
+    \---tulips
+            100930342_92e8746431_n.jpg
+            107693873_86021ac4ea_n.jpg
+            10791227_7168491604.jpg  
+```
+
+## Next steps
+Follow these tutorials to build machine learning apps with Model Builder:
+
+- [Predict prices using regression](/machine-learning/tutorials/predict-prices-with-model-builder.md)
+- [Categorize images using image classification](/machine-learning/tutorials/image-classification-api-transfer-learning.md )
```

Comment:
```suggestion
- [Analyze sentiment in a web application using binary classification](/machine-learning/tutorials/sentiment-analysis-model-builder.md )
```

Swap out image classification for sentiment analysis. Image classification tutorial is for the API, not model builder.